MissingWiki.php

```diff
@@ -1,78 +1,92 @@
-                <p style="align:center;">
-                    <table style="font-style: italic; font-size: 14px;" align="center">
-                        <tbody>
-                            <tr>
-                                <td>Host: {$_SERVER['HTTP_HOST']}</td>
-                                <td>Visitor IP: {$_SERVER['REMOTE_ADDR']}</td>
-                            </tr>
-                            <tr>
-                                <td>Request URL: {$requestURL}</td>
-                                <td>Timestamp: {$fullTimestamp}</td>
-                            </tr>
-                        </tbody>
-                    </table>
-                </p> 
-            </div>
-            <div style="float: right; padding-right: 1em;">
-                <a href="https://meta.miraheze.org/wiki/Miraheze">
-                    <img src="https://static.miraheze.org/metawiki/7/7e/Powered_by_Miraheze.png" alt="Powered by Miraheze" />
-                </a>
-            </div>
-        </body>
-    </html>
+	header( "HTTP/1.1 404 Not Found" );
+	$requestURL = htmlspecialchars( $_SERVER['REQUEST_URI'] );
+	date_default_timezone_set( 'UTC' ); // Set to UTC +0
+	$fullTimestamp = date( 'Y-m-d H:i:s' );
+	echo <<<EOF
+	<!DOCTYPE html>
+	<html lang="en">
+		<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta name="description" content="Wiki not Found">
+		<title>Wiki not Found</title>
+		<!-- Bootstrap core CSS -->
+		<link rel="stylesheet" href="https://static.miraheze.org/static/css/bootstrap.min.css">
+		<link rel="stylesheet" href="https://static.miraheze.org/static/css/font-awesome.min.css">
+		<style>
+			/* Error Page Inline Styles */
+			body {
+			  padding-top: 20px;
+			}
+			/* Layout */
+			.jumbotron {
+			  font-size: 21px;
+			  font-weight: 200;
+			  line-height: 2.1428571435;
+			  color: inherit;
+			  padding: 10px 0px;
+			}
+			/* Everything but the jumbotron gets side spacing for mobile-first views */
+			.masthead, .body-content, {
+			  padding-left: 15px;
+			  padding-right: 15px;
+			}
+			/* Main marketing message and sign up button */
+			.jumbotron {
+			  text-align: center;
+			  background-color: transparent;
+			}
+			.jumbotron .btn {
+			  font-size: 21px;
+			  padding: 14px 24px;
+			}
+			/* Colors */
+			.green {color:#5cb85c;}
+			.orange {color:#f0ad4e;}
+			.red {color:#d9534f;}
+		</style>
+		<script type="text/javascript">
+		  function loadDomain() {
+			var display = document.getElementById("display-domain");
+			display.innerHTML = document.domain;
+		  }
+		</script>
+		</head>
+		<div class="container">
+		  <!-- Jumbotron -->
+		  <div class="jumbotron">
+			<h1><img src="https://static.miraheze.org/metawiki/3/35/Miraheze_Logo.svg"> Wiki not Found</h1>
```

Comment:
could we have 404 Wiki not Found? Just to still keep the original error code in view